client/web/src/repo/settings/RepoSettingsSidebar.tsx

```diff
@@ -23,23 +27,39 @@ export const RepoSettingsSidebar: React.FunctionComponent<React.PropsWithChildre
     repoSettingsSidebarGroups,
     className,
     repo,
-}: Props) => (
-    <div className={className}>
-        {repoSettingsSidebarGroups.map(
-            ({ header, items, condition = () => true }, index) =>
-                condition({}) && (
-                    <SidebarGroup key={index}>
-                        {header && <SidebarGroupHeader label={header.label} />}
-                        {items.map(
-                            ({ label, to, exact, condition = () => true }) =>
-                                condition({}) && (
-                                    <SidebarNavItem to={`${repo.url}/-/settings${to}`} exact={exact} key={label}>
-                                        {label}
-                                    </SidebarNavItem>
-                                )
-                        )}
-                    </SidebarGroup>
-                )
-        )}
-    </div>
-)
+}: Props) => {
+    const [isMobileExpanded, setIsMobileExpanded] = useState(false)
+    const collapseMobileSidebar = useCallback((): void => setIsMobileExpanded(false), [])
```

Comment:
The return type can be inferred by TS here

```suggestion
    const collapseMobileSidebar = useCallback(() => setIsMobileExpanded(false), [])
```